### PR TITLE
fx: propogate influx deployment labels to pods

### PR DESCRIPTION
**Pull request by Freyert**
#### What this PR does / why we need it:
This PR propagates Deployment labels down to the Pod for InfluxDB. This makes it so that you can easily query for the Pods by their InfluxDB release in Prometheus and group metrics by release. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md

### CODE REVIEW

File: stable/influxdb/templates/deployment.yaml [hunk 1]
```diff
@@ -13,6 +13,9 @@ spec:
     metadata:
       labels:
         app: {{ template "influxdb.fullname" . }}
+        chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+        release: "{{ .Release.Name }}"
+        heritage: "{{ .Release.Service }}"
```
Comment (unguiculus) [inline stable/influxdb/templates/deployment.yaml]: Only add the release label.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/influxdb/templates/deployment.yaml (hunk 1 above)
Comment (Freyert) [reply stable/influxdb/templates/deployment.yaml]: Fixed! Thanks @unguiculus

**Review by unguiculus — APPROVED** [review]

**Pull request merged**